ee/tabby-webserver/Cargo.toml

```diff
@@ -7,17 +7,20 @@ homepage.workspace = true
 
 [dependencies]
 anyhow.workspace = true
+argon2 = "0.5.1"
 async-trait.workspace = true
 axum = { workspace = true, features = ["ws"] }
 bincode = "1.3.3"
 chrono = "0.4"
 futures.workspace = true
 hyper = { workspace = true, features=["client"]}
+jsonwebtoken = "9.1.0"
 juniper.workspace = true
 juniper-axum = { path = "../../crates/juniper-axum" }
 lazy_static = "1.4.0"
 mime_guess = "2.0.4"
 pin-project = "1.1.3"
+regex = "1.10.2"
```

Comment:
extract to workspace?

https://github.com/TabbyML/tabby/blob/main/Cargo.toml#L37

Reply:
Without `username` field, this create is no long needed, so removed.